diode-server/netboxdiodeplugin/client_test.go

```diff
@@ -722,6 +773,11 @@ func TestApplyChangeSet(t *testing.T) {
 				assert.Equal(t, r.Header.Get("Authorization"), fmt.Sprintf("Token %s", tt.apiKey))
 				assert.Equal(t, r.Header.Get("User-Agent"), fmt.Sprintf("%s/%s", netboxdiodeplugin.SDKName, netboxdiodeplugin.SDKVersion))
 				assert.Equal(t, r.Header.Get("Content-Type"), "application/json")
+				if tt.changeSetRequest.BranchID != "" {
+					assert.Equal(t, r.Header.Get("X-Netbox-Branch"), tt.changeSetRequest.BranchID)
+				} else {
+					assert.Len(t, r.Header.Values("X-Netbox-Branch"), 0)
+				}
```

Comment:
Can we use `netboxdiodeplugin.NetBoxBranchHeader` constant instead of hardcoded values or do we want to assert if the client implementation (with constant) against to expected header name here?

Reply:
(changed) that's fine too -- the only difference I think is that it assumes that the constants are spelled correctly (doesn't catch spelling NetBoxBranchParam `_barnch` by accident) -- but that is unlikely and easy to notice.